yarn.lock

```diff
@@ -355,6 +355,7 @@ antigravity@artsy/antigravity:
   version "0.1.3"
   resolved "https://codeload.github.com/artsy/antigravity/tar.gz/a4438d2fe9d0cdf71f1c47faba371cd3d004e140"
   dependencies:
+    coffeescript "1.11.1"
```

Comment:
so sad /cc @mzikherman

Reply:
oh wait, this is the right one

Reply:
🤔 I wonder this was not there from the beginning...